src/base64.js

```diff
@@ -7,7 +7,7 @@ function Base64() {
     this._keyMap = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=';
 
     this.isValid = function(string) {
-        return (typeof string === 'string') ? Validator.isBase64(string) : false;
+        return (typeof string === 'string') ? Validator.isBase64(string) : false; // the false one does not seem to be covered by UTs
```

Comment:
@chancehooper this is non-issue because we never use this part of the RFC specs in your code

Reply:
@martinswiderski  I would make a case for it based on the RFC just to close it off, but ones below may be more important.

Reply:
Sure sir, will do... not today though

Reply:
It's backend and not a pixel push, when? ETA please?